server.js

```diff
@@ -1633,7 +1633,100 @@ app.get('/adminAnalytics', function(req,res) {
     });
 });
 
+app.post('/submitFeedback', function(req, res){
```

Comment:
include comments on what this is used for, I know it says submitFeedback which is obvious, but include comments for code base